src/LegacyCallHandler.tsx

```diff
@@ -146,6 +147,10 @@ export enum LegacyCallHandlerEvent {
     CallState = "call_state",
 }
 
+function isNonNull<T>(thing: T | null): thing is T {
+    return thing !== null;
+}
+
```

Comment:
This will probably need to move somewhere else - going to ask about whether we have a place for typeguards